aries-site/src/pages/components/index.js

```diff
@@ -1,33 +1,59 @@
 import React from 'react';
-import { Layout, NavPage } from '../../layouts';
-import { DescriptiveHeader, Meta, ComingSoon } from '../../components';
-import { getPageDetails } from '../../utils';
+import Link from 'next/link';
+import { Box, Heading, Paragraph } from 'grommet';
+
+import { CardGrid, ContentCard, Meta } from '../../components';
+import { structure } from '../../data';
+import { Layout, PageIntro } from '../../layouts';
+import { getPageDetails, getParentPage, nameToPath } from '../../utils';
 
 const title = 'Components';
-const page = getPageDetails(title);
+const pageDetails = getPageDetails(title);
 
-const Components = () => {
-  const descriptiveHeader = (
-    <DescriptiveHeader
-      background={page.color}
-      subText={page.description}
-      icon={page.icon}
-      title={title}
-    />
-  );
+const cards = structure
+  .map(obj => {
+    const page = obj;
+    const parent = getParentPage(page.name);
+    page.parent = parent;
+    return page;
+  })
+  .filter(page => page.parent !== undefined)
+  .filter(page => page.parent.name === title);
 
+const Components = () => {
   return (
-    <Layout descriptiveHeader={descriptiveHeader} title={title} isNavPage>
+    <Layout title={title} isLanding>
       <Meta
         title={title}
-        description={page.seoDescription}
-        canonicalUrl="https://design-system.hpe.design/components"
+        description={pageDetails.seoDescription}
+        canonicalUrl="https://design-system.hpe.design/templates"
       />
-      {page.pages.length ? (
-        <NavPage items={page.pages} topic={page.name.toLowerCase()} />
-      ) : (
-        <ComingSoon />
-      )}
+      <Box gap="large">
+        <PageIntro
+          image={{
+            src: '/card-components.png',
+            alt: 'Card deck of HPE Design System component cards',
+            fit: 'cover',
+          }}
+        >
+          <Box justify="center" fill>
+            <Heading margin="none">{title}</Heading>
+            <Paragraph fill>{pageDetails.description}</Paragraph>
+          </Box>
+        </PageIntro>
+        <CardGrid>
+          {cards.map(topic => (
+            // Need to pass href because of: https://github.com/zeit/next.js/#forcing-the-link-to-expose-href-to-its-child
+            <Link key={topic.name} href={nameToPath(topic.name)} passHref>
+              <ContentCard
```

Comment:
Why aren't we treating this link directly as Anchor?  The style prop seems wrong.

Reply:
Mmmm... good question. Will re-evaluate.

Reply:
So the reasoning for this approach was to ensure an `<a>` was rendered in the DOM and is consistent with what we had previously. Applying the `<Link as={Anchor} />` results in the DOM as an `href` on a `div` instead. This was originally an issue https://github.com/hpe-design/design-system/issues/256.

I did implement re-look at the `<Link as={Anchor} />` approach, but failed tests because the tests are looking for `<a>` in the DOM. 

Part of the motivation behind ensuring the elements resolve as anchors was for web crawlers and search engines. I also looked into this but have found mixed information, however this is an answer on Google Search console: https://support.google.com/webmasters/answer/9112205?hl=en

Open to discussion/views on what we want to do here.  @britt6612 @taysea @ShimiSun

Reply:
I am fine with this implementation since it matches what we had previously been doing on the home page tiles: https://github.com/hpe-design/design-system/blob/master/aries-site/src/pages/index.js#L25

This approach was taken to resolve the issue you already mentioned and we used that google link as validation for why

Reply:
If it's helpful, we can add in the comment from the above link so the reasoning persists with these changes

Reply:
@taysea Implemented.